dali/operators/sequence/optical_flow/optical_flow.cc

```diff
@@ -109,34 +117,23 @@ void OpticalFlow<GPUBackend>::RunImpl(Workspace<GPUBackend> &ws) {
     CUDA_CALL(cudaEventRecord(sync_, ws.stream()));
     CUDA_CALL(cudaStreamWaitEvent(of_stream, sync_, 0));
   }
+  const auto &input = ws.Input<GPUBackend>(0);
+  auto &output = ws.Output<GPUBackend>(0);
+  output.SetLayout("HWC");  // Channels represent the two flow vector components (x and y)
```

Comment:
I know it was already there, but:
1. Isn't the sample_dim == 4? Shouldn't we mention the leading `F` in the output layout?
2. What happens if somebody calls the op with layout FWHC?

Reply:
Added layout info to the op schema.
Fixed layout set in the op.